aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js

```diff
@@ -17,103 +24,45 @@ const statusIcons = {
   'Ready to ship': <Package />,
 };
 
-export const FilteringLists = ({ containerRef }) => {
-  // containerRef is for demonstration purposes on this site. Most
-  // implementations should likely remove.
-
-  const size = useContext(ResponsiveContext);
-  const formFieldProps = {
-    width: !['xsmall', 'small'].includes(size) ? 'small' : undefined,
-  };
-  // Define which attributes should be made available for the user
-  // to filter upon
-  const filtersConfig = [
-    {
-      property: 'service',
-      label: 'Service',
-      filterType: 'CheckBoxGroup',
-      // contentProps accept any Grommet Box props to customize the filter's
-      // FormField. https://v2.grommet.io/formfield#contentProps
-      contentProps: formFieldProps,
-    },
-    {
-      property: 'status',
-      label: 'Status',
-      filterType: 'CheckBoxGroup',
-      contentProps: formFieldProps,
-    },
-    {
-      property: 'tenant',
-      label: 'Tenant',
-      filterType: 'CheckBoxGroup',
-      contentProps: formFieldProps,
-    },
-  ];
-
-  // Customize layer properties. Any Grommet Layer props, plus Box props
-  // for the Layer's container and contents.
-  const layerProps = {
-    // containerProps accept any Grommet Box props to customize the Box
-    // containing the layer's header, main content, and footer
-    containerProps: {
-      fill: false,
-      pad: {
-        horizontal: ['xsmall', 'small'].includes(size) ? undefined : 'medium',
-        vertical: 'small',
-      },
-      width: { max: 'large' },
-    },
-    // contentProps accept any Grommet Box props to customize the Box
-    // containing the FormField filters.
-    contentProps: {
-      direction: ['xsmall', 'small'].includes(size) ? 'column' : 'row',
-      gap: ['xsmall', 'small'].includes(size) ? 'xsmall' : 'medium',
-      pad: { horizontal: 'medium', bottom: 'small' },
-    },
-    full: ['xsmall', 'small'].includes(size),
-    // containerRef is for demonstration purposes on this site. Most
-    // implementations should likely remove.
-    target: containerRef && containerRef.current,
-    position: ['xsmall', 'small'].includes(size) ? undefined : 'center',
-  };
+// Define Data properties
+const properties = {
+  service: { label: 'Service' },
+  status: { label: 'Status' },
+  tenant: { label: 'Tenant' },
+};
 
-  return (
-    <Box
-      background="background"
-      pad={!['xsmall', 'small'].includes(size) ? 'large' : 'medium'}
-      gap="medium"
-    >
-      <Header gap="small">
+export const FilteringLists = () => (
+  <Page fill>
+    <PageContent>
```

Comment:
Unsure why the horizontal pad is eliminated in fullscreen mode...

Reply:
This is a recurring issue with Example container which will be resolved in the existing Example Refactor ticket here:  https://github.com/grommet/hpe-design-system/issues/2247